pkgs/development/python-modules/manga-ocr/default.nix

```diff
@@ -8,15 +8,14 @@ with python3Packages;
 
 buildPythonPackage rec {
   pname = "manga-ocr";
-  version = "0.1.13";
-  disabled = pythonOlder "3.7";
+  version = "0.1.14";
   format = "pyproject";
```

Comment:
We should switch that to pyproject = true

Reply:
#379092